### PR TITLE
Update adobe-ims-config.md

### PR DESCRIPTION
**Pull request by andimov**

### CODE REVIEW

File: help/getting-started/adobe-ims-config.md [hunk 1]
```diff
@@ -17,16 +17,18 @@ This integration supports Admin users who have an Adobe ID and who want to strea
 The administrator who configures this integration needs the following credentials during module enablement:
 
 * Organization ID (obtained from [Adobe Admin Console](https://adminconsole.adobe.com/)), which must be at least 24 characters in length. The authenticated user must belong to this IMS organization.
+* 2FA should be enforced on the Organization level in Adobe Admin Console to enable the module.
```
Comment (jfrontain) [inline help/getting-started/adobe-ims-config.md:20]: 2FA should be enforced on the organization level in the Adobe Admin Console when enabling the module. See [Two-step verification](https://helpx.adobe.com/enterprise/using/authentication-settings.html#two-step-verification).

Comment (jfrontain) [reply help/getting-started/adobe-ims-config.md:20]: tweaked this one line and added a link

**Comment by leslietilling** [comment]
@andimov  You must complete the CLA to submit through this external repo. It is strongly recommended for Adobe employees to submit PRs in the internal repository (https://git.corp.adobe.com/AdobeDocs/commerce-admin.en).

**Comment by andimov** [comment]
@leslietilling I'm closing this PR as I've created an internal one https://git.corp.adobe.com/AdobeDocs/commerce-admin.en/pull/29
could you please review it?

**Pull request closed without merging**